sigal/writer.py

```diff
@@ -81,6 +83,9 @@ def __init__(self, settings, index_title=''):
             **env_options
         )
 
+        for filter in filters:
+            env.filters[filter.__name__] = filter
```

Comment:
To be sure we are talking about the same thing: you mean permit user to write their own filters in `sigal.conf.py` and then apply it in their template ?

As filters are heavily linked to the templates, shouldn't it be better to permit custom filters in templates directory ? For example, in a template directory, next to the `templates` and `static` folder, one can define a `filters.py` who whould contains all needed filters for this template ?